src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonNumberHandling.cs

```diff
@@ -23,9 +23,9 @@ public enum JsonNumberHandling
         /// </summary>
         WriteAsString = 0x2,
         /// <summary>
-        /// The "NaN", "Infinity", and "-Infinity" <see cref="JsonTokenType.String"/> tokens can be read as floating-point constants,
-        /// and the <see cref="float.NaN"/>, <see cref="double.PositiveInfinity"/>, and <see cref="float.NegativeInfinity"/>
-        /// values will be written as their corresponding JSON string representations.
+        /// The "NaN", "Infinity", and "-Infinity" <see cref="JsonTokenType.String"/> tokens can be read as
+        /// floating-point constants, and the <see cref="float"/> and <see cref="double"/> values for these
+        /// constants will be written as their corresponding JSON string representations.
```

Comment:
nit: I think having example values might be useful.
```suggestion
        /// The "NaN", "Infinity", and "-Infinity" <see cref="JsonTokenType.String"/> tokens can be read as
        /// floating-point constants, and the <see cref="float"/> and <see cref="double"/> values for these
        /// constants (such as <see cref="float.PositiveInfinity"/> and <see cref="double.PositiveInfinity"/>) will be written as their corresponding JSON string representations.
```